### PR TITLE
Pull request for Issue2335: Visualization for endstation kill switch status

### PR DESCRIPTION
**Pull request by mButlerDM**
Added a control for the end station kill switch in both BioXAS side and main. Added instances, connections and getters for the PV instances within their beamline classes. Added a control editor for the persistent view to view kill switch status. 

Testing complete BioXAS side, awaiting testing for main.

### CODE REVIEW

**Comment by helfrij** [comment]
Hey @iainworkman, would you review?

File: source/beamline/BioXAS/BioXASMainBeamline.cpp [hunk 1]
```diff
@@ -303,6 +304,11 @@ void BioXASMainBeamline::setupComponents()
 	beamStatus_->addComponent(m1Mirror_->mask()->state(), BioXASM1MirrorMaskState::Open);
 	beamStatus_->addComponent(mono_->maskState(), BioXASSSRLMonochromatorMaskState::Open);
 
+        // Kill switch status.
+
```
Comment (iainworkman) [inline source/beamline/BioXAS/BioXASMainBeamline.cpp]: The indentation below looks a little funny.

**Comment by iainworkman** [comment]
Looks good to me.
Time Estimate: 1 minute

**Comment by dretrex** [comment]
Looks good.
Time Estimate: 1 minute

**Pull request merged**